core/core-impl/src/main/resources/solver.xsd

```diff
@@ -1435,6 +1435,18 @@
       
   </xs:complexType>
     
+  <xs:complexType name="solverConfigOverride">
```

Comment:
If done correctly, there will be no changes to this file. See other comments on this PR.

Reply:
Yes, I confirmed that no changes have been made to this file.